oak_crypto/src/aead.rs

```diff
@@ -34,7 +34,7 @@ pub(crate) type AeadNonce = [u8; AEAD_NONCE_SIZE_BYTES];
 
 /// Encrypts `plaintext` with associated data using AES-GCM encryption scheme.
 /// Note: the corresponding associated data is NOT encrypted.
-pub(crate) fn encrypt(
+pub(crate) fn _encrypt(
```

Comment:
Why was this (and others) renamed?

Reply:
Because they were producing `unused function` errors